### PR TITLE
upgrade instructions and changelog

### PR DESCRIPTION
**Pull request by Scarjit**
This pr adds the changelog for UMH 0.13.3 (0.2.15)

### CODE REVIEW

File: umh.docs.umh.app/content/en/docs/production-guide/upgrading/0.13.3.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+```bash
+sudo $(which helm) repo update --kubeconfig /etc/rancher/k3s/k3s.yaml
+```
+
+
+## Upgrade Helm Chart
+
+Upgrade the Helm chart to the 0.13.4 version:
+```
+sudo $(which helm) upgrade united-manufacturing-hub united-manufacturing-hub/united-manufacturing-hub -n united-manufacturing-hub --version 0.13.4 --reuse-values --kubeconfig /etc/rancher/k3s/k3s.yaml \
```
Comment (JeremyTheocharis) [inline umh.docs.umh.app/content/en/docs/production-guide/upgrading/0.13.3.md:28]: no schema migrations?

**Review by JeremyTheocharis — CHANGES_REQUESTED** [review]

**Comment by Scarjit** [comment]
@JeremyTheocharis The migration is quite long.
We could put it into a bash script and do a `bash <(curl -s https://umh.docs.umh.app/path/to/script.sh)`

**Comment by Scarjit** [comment]
It would also allow a bit cleaner handling of already existing stuff like users and databases

**Comment by JeremyTheocharis** [comment]
What about https://management.umh.app/upgrading/upgrading-to-0.95.3.sh ?

**Comment by Scarjit** [comment]
image

Im now creating the redirect

**Review by JeremyTheocharis — APPROVED** [review]

**Review by JeremyTheocharis — CHANGES_REQUESTED** [review]
would this not make more sense in the managementconsole repo?

**Pull request merged**